src/pages/resource/ResourceDescriptionForm.tsx

```diff
@@ -0,0 +1,164 @@
+                    />
+                  </div>
+                  <div className="field-wrapper">
+                    <Field
+                      name="description"
+                      label={t('resource_form.Description')}
+                      component={TextField}
+                      multiline
+                      rows="4"
+                      fullWidth
+                      variant="outlined"
+                    />
+                  </div>
+                  <div className="multiple-field-wrapper ">
+                    <div className="field-wrapper ">
+                      <Field
+                        name="NPI"
+                        label={t('resource_form.NPI')}
+                        component={TextField}
+                        variant="outlined"
+                        fullWidth
+                      />
+                    </div>
+                    <div className="field-wrapper">
+                      <Field
+                        name="keyword"
+                        label={t('resource_form.Tags')}
+                        component={TextField}
+                        fullWidth
+                        variant="outlined"
+                      />
+                    </div>
+                  </div>
+
+                  <div className="multiple-field-wrapper ">
+                    <div className="field-wrapper ">
+                      <Field component={FormikDatePicker} name="date" />
+                    </div>
+
+                    <div className="field-wrapper">
+                      <Field name="language" variant="outlined" fullWidth component={Select} label={t('date')}>
+                        {languages.map(language => (
+                          <MenuItem
+                            value={language.code}
+                            key={language.code}
+                            data-cy={`user-language-${language.code}`}>
+                            {language.name}
+                          </MenuItem>
+                        ))}
+                      </Field>
+                    </div>
+                  </div>
+
+                  <div className="header">{t('resource_form.project_assosiation')}</div>
+
+                  <div className="field-wrapper">
+                    <Field
+                      name="project"
+                      label={t('resource_form.Project')}
+                      component={Select}
+                      fullWidth
+                      variant="outlined">
+                      {publications.map(publication => (
+                        <MenuItem value={publication.id} id="pub-item" key={publication.id}>
+                          {`${publication.name} - ${publication.id}`}
+                        </MenuItem>
+                      ))}
+                    </Field>
+                  </div>
+                  <Button type="submit" onClick={(_: any) => handleValidation(values)}>
+                    Validate
```

Comment:
Virker som at `type="submit"` trigger `onSubmit` også by default.

Reply:
det stemmer

Reply:
also: i18n for "Validate"

Reply:
lar den stå siden vi neppe skal ha knappen der senere... Skjema bør validere på onBlur, men ville ikke slette koden